### ISSUE TITLE
[Feature] Sum up allocation sub classes

### ISSUE THREAD

**Issue by putzwasser** [redstreet/fava_investor#75]
Currently fava_investor shows the percentage for each asset class in relation to the total:
![image](https://user-images.githubusercontent.com/26040044/201131379-b9cc3751-fc40-4bb2-9fa0-8ef52a4322c8.png)

It would be nice if fava_investor showed the sum of a class in regards to the total and to its parent(s):
![image](https://user-images.githubusercontent.com/26040044/201132667-d577efc6-7250-4ad5-a8c8-f23d181e7eac.png)
(Something like this)

**Comment by redstreet** [comment]
It already does: hover your mouse over the percentage. Give it a shot and let me know.

**Comment by putzwasser** [comment]
Ah, I see :+1: 

May I suggest some improvements:

- Displaying the amounts/percentages on hover is unintuitive and people probably will never/not know this piece of informatin exists (I'm the best example :laughing: )
- Having to hover over the rows to see the data makes it very difficult to compare allocations, e.g., to see if you're off your target allocation.
- Therefore, I'd favor a solution where the data is either always present or visible for all rows after a single click (key is best UX, IMHO).*

\* I noticed that the data shows if you collapse the "subrows", but then again, part of the information is missing.

Say, your target domestic stock allocation is 45%. First, you need to collapse the `domestic` row. Then you'd see your current allocation is `50.12%`. Then, you'd be wondering: “Which position would I want to decrase?” One more click to expand `domestic` again, to see the allocation within it. “Is `largecap` as big *within* `domestic` as I think?” => Hover above `largecap`.

I mean, this works. But it is cumbersome in my opinion.

Please don't get me wrong. I don't want to be ungrateful for the work you've done. I really enjoy it. Just wanting to contribute to better data accessibility/visibility.

**Comment by redstreet** [comment]
I agree that this would be a nice to have. The way people use AA is very individual, and so, this needs to be thought out well through a broader set of use cases. I just opened #78 for related higher level functionality.

There are many good ideas here, but the bottleneck is contribution time. Volunteers would help, since I'm unlikely to get to this soon. Feel free to submit a patch.
